design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -29,11 +29,6 @@ type Mutation {
         input: RequestNegotiableQuoteInput!
     ): RequestNegotiableQuoteOutput @doc(description: "Request a new negotiable quote for a buyer")
 
-    # Covers "Add your Comment" section of https://docs.magento.com/user-guide/customers/account-dashboard-quotes-negotiate.html#shipping-information
```

Comment:
since we have an alternative for this in https://github.com/magento/architecture/pull/486/files#diff-7df827613bd49741c4fecea764fff6242c93851ac71b6eb7f870dc6e88f7132aR63
then removing addNegotiableQuoteComment is the right approach